code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -360,6 +360,12 @@
 				H.apply_damage(5, BRUTE, picked_def_zone)
 				H.Weaken(3)
 				if(cooldown < world.time - 10) //cooldown to avoid message spam.
-					H.visible_message("<span class='danger'>[H] steps in the broken glass!</span>", \
-							"<span class='userdanger'>You step in the broken glass!</span>")
-					cooldown = world.time
+					if(!H.incapacitated())
+						H.visible_message("<span class='danger'>[H] steps in the broken glass!</span>", \
+								"<span class='userdanger'>You step in the broken glass!</span>")
+						cooldown = world.time
+					else
+						H.visible_message("<span class='danger'>[H] slides on the broken glass!</span>", \
+								"<span class='userdanger'>You slide on the broken glass!</span>")
+
+						cooldown = world.time
```

Comment:
@optimumtact oh fuck thanks im an autist

i could swear i was looking at that and sayin "there is something fukken wrong" and just prd it